### PR TITLE
Fixing all ACM bugs

### LINKED ISSUE #56: ACM Bugs
Bugs from incorrect rebase:

- Path issues
- Missing/Invalid CSS

### LINKED ISSUE #57: Fix 'The property -ms-transform is compatible with -webkit-transform and should be included as well.' issue in public\ACM\css\ACM.css
[CodeFactor](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/overview/acm) found an issue: The property -ms-transform is compatible with -webkit-transform and should be included as well.

It's currently on:
[public\ACM\css\ACM.css:73
](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/source/acm/public/ACM/css/ACM.css#L73)

### LINKED ISSUE #58: Fix 'Missing standard property 'transform' to go along with '-webkit-transform'
[CodeFactor](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/overview/acm) found an issue: Missing standard property &#39;transform&#39; to go along with &#39;-webkit-transform&#39;.

It's currently on:
[public\ACM\css\ACM.css:106
](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/source/acm/public/ACM/css/ACM.css#L106)

### LINKED ISSUE #59: Fix 'Missing standard property 'transform' to go along with '-webkit-transform'
[CodeFactor](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/overview/acm) found an issue: Missing standard property &#39;transform&#39; to go along with &#39;-webkit-transform&#39;.

It's currently on:
[public\ACM\css\ACM.css:103
](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/source/acm/public/ACM/css/ACM.css#L103)

### LINKED ISSUE #60: Fix 'Missing standard property 'transform' to go along with '-webkit-transform'
[CodeFactor](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/overview/acm) found an issue: Missing standard property &#39;transform&#39; to go along with &#39;-webkit-transform&#39;.

It's currently on:
[public\ACM\css\ACM.css:100
](https://www.codefactor.io/repository/github/aravindvnair99/face-amrita-bengaluru/source/acm/public/ACM/css/ACM.css#L100)

### PR DESCRIPTION
**Pull request by VinayakTekade**
Fixes #56, fixes #57, fixes #58, fixes #59, fixes #60 

- ACM.ejs and ACM.css taken from commit 0a95977 

<a href="https://gitpod.io/#https://github.com/aravindvnair99/FACE-Amrita-Bengaluru/pull/55"></a>

### CODE REVIEW

**Comment by VinayakTekade** [comment]
Don't close this pull request yet.  This will include all the bug fixes.

**Comment by aravindvnair99** [comment]
> Don't close this pull request yet. This will include all the bug fixes.

@VinayakTekade Yep, just reviewing for now. Let me know here when it's ready to be merged. Until then, I shall keep reviewing along with the bots.

**Comment by VinayakTekade** [comment]
The commits are signed off. Why is DCO failing?

**Comment by aravindvnair99** [comment]
> The commits are signed off. Why is DCO failing?

@VinayakTekade Fixed it.

File: functions/views/ACM/ACM.ejs [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 	<!-- Custom CSS -->
 	<link rel="stylesheet" href="/ACM/css/ACM.css" />
-	<title>ACM Amrita Bengaluru</title>
+	<title>ACM ASEB</title>
```
Comment (aravindvnair99) [inline functions/views/ACM/ACM.ejs]:
```suggestion
	<title>ACM Amrita Bengaluru</title>
```

For SEO

**Comment by VinayakTekade** [comment]
> > The commits are signed off. Why is DCO failing?
> 
> @VinayakTekade Fixed it.

How did you fix it? It's happening again.

File: functions/views/ACM/ACM.ejs [hunk 2]
```diff
@@ -129,7 +129,7 @@
 			<div class="card bg-light mb-3">
 				<div class="row no-gutters">
 					<div class="col-md-6 d-flex align-items-center order-1 order-md-0">
-						<img src="/ACM/img/ACM.png" class="card-img p-5" alt="ACM logo" />
+						<img src="/ACM/img/acm.png" class="card-img p-5" alt="ACM logo" />
```
Comment (aravindvnair99) [inline functions/views/ACM/ACM.ejs]:
```suggestion
						<img src="/ACM/img/ACM.png" class="card-img p-5" alt="ACM logo" />
```

**Comment by aravindvnair99** [comment]
> > > The commits are signed off. Why is DCO failing?
> > 
> > 
> > @VinayakTekade Fixed it.
> 
> How did you fix it? It's happening again.

@VinayakTekade It's proper now. I signed and pushed it.

**Comment by aravindvnair99** [comment]
@VinayakTekade Include the fix for #57 also in this PR.

**Comment by aravindvnair99** [comment]
@VinayakTekade Include the fix for #57 #58 #59 #60 also in this PR.

**Comment by VinayakTekade** [comment]
Ignore CodeFactor issues in ACM_Members for now. Will fix it when all the members are added.

**Comment by aravindvnair99** [comment]
> Ignore CodeFactor issues in ACM_Members for now. Will fix it when all the members are added.

#63

**Pull request merged**